src/m2ee/version.py

```diff
@@ -41,7 +41,7 @@ def __str__(self):
         return version
 
     def __contains__(self, other):
-        if isinstance(other, basestring):
+        if isinstance(other, str):
```

Comment:
str vs. basestring: https://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility

Including this seems most python2/3 compatible:

    try:
        basestring
    except NameError:
        basestring = str

Reply:
I think this only breaks when doing `__future__ import unicode_literals`, but yes, I agree. OTOH, why not simply reverse the check, `if not isinstance(other, MXVersion)`?

Reply:
Hm, yes, the reverse is probably simpler. Added in #33 :)